cpp/include/cudf/table/experimental/row_operators.cuh

```diff
@@ -75,6 +75,19 @@ namespace row {
 enum class lhs_index_type : size_type {};
 enum class rhs_index_type : size_type {};
 
+/**
+ * @brief A counting iterator that uses strongly-typed indices bound to tables.
+ *
+ * Performing lexicographic or equality comparisons between values in two
+ * tables requires the use of strongly-typed indices. The strong index types
```

Comment:
> Compounds formed by an adverb ending in ly plus an adjective or participle (such as largely irrelevant or smartly dressed) are not hyphenated either before or after a noun, since ambiguity is virtually impossible. (The ly ending with adverbs signals to the reader that the next word will be another modifier, not a noun.) —Chicago Manual of Style, 7.82.
https://www.dailywritingtips.com/adverbs-and-hyphens/

```suggestion
 * @brief A counting iterator that uses strongly typed indices bound to tables.
 *
 * Performing lexicographic or equality comparisons between values in two
 * tables requires the use of strongly typed indices. The strong index types
```